x-pack/legacy/plugins/uptime/public/components/functional/ping_list.tsx

```diff
@@ -253,22 +254,17 @@ export const PingListComponent = ({
                         defaultMessage: 'Status',
                       })}
                     >
-                      <EuiComboBox
-                        isClearable={false}
-                        singleSelection={{ asPlainText: true }}
-                        selectedOptions={[
-                          statusOptions.find(({ value }) => value === selectedOption) ||
-                            statusOptions[2],
-                        ]}
+                      <EuiSelect
                         options={statusOptions}
                         aria-label={i18n.translate('xpack.uptime.pingList.statusLabel', {
                           defaultMessage: 'Status',
                         })}
-                        onChange={(selectedOptions: EuiComboBoxOptionProps[]) => {
-                          if (typeof selectedOptions[0].value === 'string') {
+                        value={selectedOption}
+                        onChange={(selected) => {
+                          if (typeof selected.target.value === 'string') {
                             onSelectedStatusChange(
                               // @ts-ignore it's definitely a string
-                              selectedOptions[0].value !== '' ? selectedOptions[0].value : null
+                              selected.target !== '' ? selected.target.value : null
```

Comment:
`selected.target` is type `EventTarget & HTMLSelectElement`, so it's never going to be equal to `''`. This type mismatch was concealed by my old `@ts-ignore` comment, which we can safely decommission.

```suggestion
                              !!selected.target && selected.target.value !== ''
                                ? selected.target.value
                                : null
```